com.yakindu.sct.examples.basic.multism/model/controller.sct

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<xmi:XMI xmi:version="2.0" xmlns:xmi="http://www.omg.org/XMI" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:notation="http://www.eclipse.org/gmf/runtime/1.0.2/notation" xmlns:sgraph="http://www.yakindu.org/sct/sgraph/2.0.0">
-  <sgraph:Statechart xmi:id="_z7pGgF1bEeqOhZ2ZqWiu-Q" specification="@EventDriven&#xA;@SuperSteps(no)&#xA;&#xA;import: &quot;led.sct&quot;&#xA;&#xA;interface:&#xA;&#x9;var green : led&#xA;&#x9;var red : led&#xA;&#xA;&#xA;internal:&#xA;&#x9;entry / &#xA;&#x9;&#x9;green.enter;&#xA;&#x9;&#x9;red.enter" name="controller">
+<xmi:XMI xmi:version="2.0" xmlns:xmi="http://www.omg.org/XMI" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:notation="http://www.eclipse.org/gmf/runtime/1.0.3/notation" xmlns:sgraph="http://www.yakindu.org/sct/sgraph/2.0.0">
```

Comment:
`gmf/runtime/1.0.3/notation` this will give us problems when used in installations from update sites into older eclipses (?)

Reply:
this happens if you're working on with 2020-06 and save the models.. I'll update all of them to 1.0.2